entities/src/main/java/org/odk/collect/entities/javarosa/filter/PullDataFunctionHandler.kt

```diff
@@ -0,0 +1,49 @@
+package org.odk.collect.entities.javarosa.filter
+
+import org.javarosa.core.model.condition.EvaluationContext
+import org.javarosa.core.model.condition.IFunctionHandler
+import org.javarosa.xpath.expr.XPathFuncExpr
+import org.odk.collect.entities.javarosa.intance.LocalEntitiesInstanceAdapter
+import org.odk.collect.entities.storage.EntitiesRepository
+
+class PullDataFunctionHandler(
+    entitiesRepository: EntitiesRepository,
+    private val fallback: IFunctionHandler? = null
+) : IFunctionHandler {
+
+    private val instanceAdapter = LocalEntitiesInstanceAdapter(entitiesRepository)
+
+    override fun getName(): String {
+        return NAME
+    }
+
+    override fun getPrototypes(): List<Array<Class<Any>>> {
+        return emptyList()
+    }
+
+    override fun rawArgs(): Boolean {
+        return true
+    }
+
+    override fun realTime(): Boolean {
+        return false
+    }
+
+    override fun eval(args: Array<Any>, ec: EvaluationContext): Any {
+        val instanceId = XPathFuncExpr.toString(args[0])
```

Comment:
The existing `ExternalDataHandlerPull` normalizes this by removing `.csv` suffix. Should we do the same here to make the behavior consistent? See: https://github.com/getodk/collect/blob/master/collect_app/src/main/java/org/odk/collect/android/dynamicpreload/handler/ExternalDataHandlerPull.java#L83